### DISCUSSION TITLE
How can I view the changes made by me locally?

### DISCUSSION THREAD

**Discussion by AmrutaKoshe** [Praful932/Kitabe#92] [Q&A]
I am not able to check the changes that I have made. 

This is the original code in this repo, where the colours in more-details class are blue and white 
![image](https://user-images.githubusercontent.com/59871941/111057906-d1914080-84b0-11eb-9fd9-b92aa61798ef.png)

I made some changes and changed the colours to green and red, and saved the changes
![image](https://user-images.githubusercontent.com/59871941/111057912-dfdf5c80-84b0-11eb-9bb3-e4dd7b40ed62.png)

After refreshing the webpage, the changes are not being reflected. The "More details" button is still blue 😅
![image](https://user-images.githubusercontent.com/59871941/111058016-93e0e780-84b1-11eb-8a88-00c55c1122b0.png)

**Comment by maneprajakta** [comment]
Try to clear the cache by ctrl+r

**Reply by AmrutaKoshe** [reply]
It worked! Thanks a lot ^^

**Reply by maneprajakta** [reply]
There can be many reasons :)
